Remove "Account Settings" from the status menu in the top right of the screen
since we aren't shipping the gnome-user-accounts-panel on Solaris due to
../gnome-control-center/patches/01-user-accounts.patch .

--- a/js/ui/status/system.js
+++ b/js/ui/status/system.js
@@ -222,8 +222,9 @@
                                           'visible',
                                           bindFlags);
 
-        this._switchUserSubMenu.menu.addSettingsAction(_("Account Settings"),
-                                                       'gnome-user-accounts-panel.desktop');
+// gnome-user-accounts-panel is not currently available on Solaris
+//        this._switchUserSubMenu.menu.addSettingsAction(_("Account Settings"),
+//                                                       'gnome-user-accounts-panel.desktop');
 
         this._user.connect('notify::is-loaded', this._updateSwitchUserSubMenu.bind(this));
         this._user.connect('changed', this._updateSwitchUserSubMenu.bind(this));
